### ISSUE TITLE
Track and possibly port a patch for inclusion

### ISSUE THREAD

**Issue by emilyst** [emilyst/hid-nx-dkms#5]
https://git.kernel.org/pub/scm/linux/kernel/git/hid/hid.git/commit/?h=for-6.1/nintendo&id=1ff89e06c2e5fab30274e4b02360d4241d6e605e

**Comment by jpwhiting** [comment]
Looks like that one landed at least in hid-nintendo.c in the kernel (from what I'm seeing in a 6.2.2 arch kernel sources here) not sure if it landed in hid-nx though, looking.

**Comment by jpwhiting** [comment]
Looks to not be in hid-nx.c yet. Ignore my noise.

**Comment by emilyst** [comment]
Yeah, I just need to find time to do this. I was unemployed last year, so I had more energy for these sorts of things.

**Comment by jpwhiting** [comment]
Yep, no worries. Enjoying this so far. Are there plans to get this into mainline at some point in the short or long terms?

**Comment by emilyst** [comment]
In the short term: no, nothing’s planned. I’m pretty busy with my day job.

In the long term: I’ve wanted to, but there are some issues. I’ve done some relatively invasive refactoring that makes my changes difficult simply to drop in place. I also just don’t really know the process, or how welcome such changes would be, so I am guessing I would have to backport my changes to slot into the mainline module less intrusively.

That essentially means rewriting much of what I’ve done here, with all that implies for stability.
